src/styles/footer.scss

```diff
@@ -1,28 +1,44 @@
-footer {
-    padding: 4.8rem 0rem 5rem 0rem;
-    font-family: "Roboto", sans-serif;
-    display:flex;
-    flex-direction: column;
-    justify-content: space-evenly;
-    align-items: center;
-    a {
-        color: black;
-        text-decoration: none;
-        &:hover {
-            color: #4ca1af;
-        }
+footer.page-footer {
```

Comment:
Since your footer component appears on every page, this stylesheet will appear on every page, and any other `<footer>` element would receive these styles. Adding a class better targets _this_ specific footer.